cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/GoalOptimizer.java

```diff
@@ -143,11 +143,11 @@ public void run() {
       if (loadMonitorTaskRunnerState == LOADING || loadMonitorTaskRunnerState == BOOTSTRAPPING) {
         LOG.info("Skipping proposal precomputing because load monitor is in " + loadMonitorTaskRunnerState + " state.");
         // Check in 30 seconds to see if the load monitor state has changed.
-        sleepTime = 30000L;
+        sleepTime = TimeUnit.SECONDS.toMillis(30);
       } else if (!_loadMonitor.meetCompletenessRequirements(_requirementsWithAvailableValidWindows)) {
         LOG.info("Skipping proposal precomputing because load monitor does not have enough snapshots.");
         // Check in 30 seconds to see if the load monitor has sufficient number of snapshots.
-        sleepTime = 30000L;
+        sleepTime = TimeUnit.SECONDS.toMillis(30);
       } else {
         try {
           if (!validCachedProposal()) {
```

Comment:
Nit (multiple occurrences in this class): Can we
1. Move the hardcoded `TimeUnit.SECONDS.toMillis(30)` value to some static variable?
2. Update the comments such as `Check in 30 seconds to see...` to avoid mentioning hardcoded times and refer to the static variable?

Reply:
I used `{@value HALF_MINUTE_IN_MS}` to replace `30 seconds` in comments